public/index.php

```diff
@@ -73,12 +73,22 @@
   // "callback" => function ($request, $response, $arguments) use ($container) { ???
   "before" => function ($request, $arguments)
   {
-    $myUser = \App\v1\Models\User::find($arguments['decoded']['user_id']);
-    if ($myUser['jwtid'] != $arguments['decoded']['jti'])
+    $myUser = \App\v1\Models\Item::find($arguments['decoded']['user_id']);
+    $jwtid = '';
+    foreach ($myUser->properties()->get() as $test)
+    {
+      if ($test->internalname == 'userjwtid')
+      {
+        $jwtid = $test->value;
+        break;
+      }
+    }
```

Comment:
Not possible in model with something like properties_by_internalname because we use Lazy Eager Loading.
I have set a function for loop and return the value ;)